### PR TITLE
Fix drag and drop of strings

### LINKED ISSUE #3160: 'Drag to upload file' overlay gets stuck and makes channel unusable
Your Rocket.Chat version: 1.3.1
Chrome: 50.0.2661.94 m
Windows 7 Professional

Often times while I'm reading something that somebody wrote, I will use my cursor to highlight what they are saying. I can get pretty "clicky" when doing this. If you click on a highlighted segment of words and then click and drag, you will get the 'Drag to upload file' overlay. Often times (like 10 times throughout the day) this overlay gets stuck and disables the current channel/direct message I'm currently in. It requires shutting down the app and restarting the app.

This has happened to me in both Chrome and the native Rocket.Chat application. I'm mainly using the app though so that's where it happens most to me.

image

### PR DESCRIPTION
**Pull request by marceloschmidt**
@RocketChat/core 

Closes #3160

### CODE REVIEW

**Comment by rodrigok** [comment]
LGTM

**Comment by geekgonecrazy** [comment]
lgtm 👍

File: packages/rocketchat-ui-sidenav/side-nav/sideNav.coffee [hunk 1]
```diff
@@ -51,6 +51,9 @@ Template.sideNav.events
 	'scroll .rooms-list': ->
 		menu.updateUnreadBars()
 
+	'dropped .side-nav': (e) ->
+		e.preventDefault()
```
Comment (marceloschmidt) [inline packages/rocketchat-ui-sidenav/side-nav/sideNav.coffee]: @sampaiodiego per your request :)

Comment (sampaiodiego) [inline packages/rocketchat-ui-sidenav/side-nav/sideNav.coffee]: yay!!!

**Pull request merged**